src/routes/_actions/delete.js

```
@@ -6,11 +6,13 @@ import { deleteStatus as deleteStatusLocally } from './deleteStatuses'
 export async function doDeleteStatus (statusId) {
   let { currentInstance, accessToken } = store.get()
   try {
-    await deleteStatus(currentInstance, accessToken, statusId)
+    let deletedStatus = await deleteStatus(currentInstance, accessToken, statusId)
     deleteStatusLocally(currentInstance, statusId)
     toast.say('Status deleted.')
+    return deletedStatus
   } catch (e) {
     console.error(e)
     toast.say('Unable to delete status: ' + (e.message || ''))
+    return {}
```

I'd rather throw the error here so that the compose dialog is not shown